package.json

```diff
@@ -131,15 +132,16 @@
   },
   "dependencies": {
     "bluebird": "^3.5.1",
-    "clappr": "0.2.78",
+    "clappr": "^0.2.85",
```

Comment:
Upgraded clappr w/ no issues :). We couldn't do this before the controls rewrite